assets/styles/tools/_functions.scss

```
@@ -139,3 +139,50 @@
 }
 
 $context: 'frontend' !default;
+
+
+
+/**
+ * Calculate grid space
+ * @param  {number} $number The percentage spacer
+ * @return {number} in calc
+ */
+@function grid-space($percentage, $offset: 0) {
+    // @return calc(#{$percentage} * (100vw - 2 * var(--grid-margin)) - (1 - #{$percentage}) * var(--grid-gutter) + #{$offset} * var(--grid-gutter));
+    @return calc(#{$percentage} * (100vw - 2 * var(--grid-margin, 0px)) - (1 - #{$percentage}) * var(--grid-gutter, 0px) + #{$offset} * var(--grid-gutter, 0px));
+}
+
+
+/**
+ * Calculate vh percentage from number
+ * @param  {number} $number The percentage number
+ * @return {number} in vh
+ */
+@function vh($number) {
+    @return calc(#{$number} * var(--vh, 1vh));
+}
```

Improve documentation, use Sass comment syntax since this is a Sass function, and have parameter name match purpose.

```suggestion
// Returns calculation of a percentage of the viewport height.
//
// ```scss
// .box {
//     height: vh(100);
// }
// ```
//
// @param  {number} $percentage
// @return {function<number>}

@function vh($percentage) {
    @return calc(#{$percentage} * var(--vh, 1vh));
}
```